.github/workflows/ci.yml

```diff
@@ -250,3 +251,20 @@ jobs:
           asset_path: ./${{ env.ZIP_FILE_NAME }}
           asset_name: ${{ env.ZIP_FILE_NAME }}
           asset_content_type: application/gzipa
+
+  post-release-checks:
+    name: Try fuelup install with fuelup-init
+    strategy:
+      matrix:
+        job:
+          - os: ubuntu-latest
+          - os: macos-latest
+    runs-on: ${{ matrix.job.os }}
+    if: github.event_name == 'release' && github.event.action == 'published'
+    needs: [cancel-previous-runs, build-release]
```

Comment:
Move these two number `name` so it's easier to see the conditions this job runs under.